.github/pull_request_template.md

```diff
@@ -9,11 +9,22 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+If some steps are not applicable or in your judgment not necessary, remove and explain why rather than leave checkmarks blank.
+
+- [ ] I've tested the code as an end user would and included appropriate testing steps 
+- [ ] For all new functionality, I have added thorough, applicable tests that validate my changes
+- [ ] For improvements to existing functionality, I've followed the [campsite rule](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git), leaving it a little better than I found it. 
 - [ ] The added feature contains necessary observability
 - [ ] I have made corresponding changes to the documentation
 
+When done with the checklist, request a review from a [merger](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git) (list available [here](https://github.com/orgs/encodium/teams/mergers))
```

Comment:
As @drpebcak mentioned https://github.com/encodium/.github-private/pull/10/files#r984348302 
It's my understanding that, in general, the Pull Request would be reviewed by peers first, then go to the 'merger', when ready, for final approval.